builder/edition_utilities.livecodescript

```diff
@@ -0,0 +1,58 @@
+﻿script "EditionUtilities"
+constant kEditionNames="community,communityplus,indy,business,commercial"
+function editionNames
+   return kEditionNames
+end editionNames
+
+function editionTitleCase pEdition
+	switch pEdition
+	case "community"
+		return "Community"
+	case "communityplus"
+		return "CommunityPlus"
+	case "indy"
+		return "Indy"
+	case "business"
+		return "Business"
+	case "commercial"
+		return "Commercial"
+	end switch
+end editionTitleCase
+
+function editionDisplayName pEdition
+	switch pEdition
+	case "community"
+		return "Community"
+	case "communityplus"
+		return "Community Plus"
+	case "indy"
+		return "Indy"
+	case "business"
+		return "Business"
+	case "commercial"
+		return "Commercial"
+	end switch
+end editionDisplayName
+
+
+function editionCompare pA, pB
+	local tA, tB
+	put itemOffset(pA, kEditionNames) into tA
```

Comment:
@livecodeian might be safer to add `set the wholeMatches to true` here incase we ever add something like `business lite` before `business`

Reply:
Good idea